provider/postgis/util_internal_test.go

```diff
@@ -7,8 +7,8 @@ import (
 
 	"github.com/jackc/pgx"
 
+	"github.com/go-spatial/proj"
```

Comment:
I know it's functionally not different, but please revert this import to the `tegola/proj` package so we're consistent.